### DISCUSSION TITLE
Can't see protocol of "Blocked adult websites","Blocked malware/phishing" and "rewritten"

### DISCUSSION THREAD

**Discussion by ElBarto2022** [AdguardTeam/AdGuardHome#4576] [Q&A]
If i try to open the logs of "Blocked adult websites","Blocked malware/phishing" and "rewritten" over the webinterface i only see the circle going round and nothing else. I use the actual version on a pfsense with FreeBSD. (Querylog.json has 24MB, sessions.db has 32KB and stats.db has 64KB)

**Comment by ainar-g** [comment]
Hello.  Are there any queries that were blocked at all?  That is, do you see them on the dashboard?

24 MiB isn't a lot, and unless the disk i/o is very slow, it should scan the log relatively quickly.  You can also [enable verbose logs][1] to see if there are errors while reading the log.

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog

**Reply by ElBarto2022** [reply]
The dashboard shows that there are 2 entries in "Blocked adult websites". I think it is a database error. For example the "Blocked by filters" category has over 20000 entries and i can display them without problems. I enabled the verbose log and see only entries likes this..

2022/05/19 17:53:42.916517 66027#118 [debug] GET /control/querylog?search=&response_status=blocked_parental&older_than=2022-05-19T13%3A28%3A18.895833554%2B0>
2022/05/19 17:53:42.916517 66027#118 [debug] QueryLog: prepared data (0/541) older than 2022-05-19 13:28:18.895833554 +0200 CEST in 0s
2022/05/19 17:53:42.916517 66027#118 [debug] GET /control/querylog?search=&response_status=blocked_parental&older_than=2022-05-19T13%3A28%3A18.895833554%2B0>
2022/05/19 17:53:42.916517 66027#118 [debug] QueryLog: prepared data (0/541) older than 2022-05-19 13:28:18.895833554 +0200 CEST in 0s
2022/05/19 17:53:42.916517 66027#118 [debug] GET /control/querylog?search=&response_status=blocked_parental&older_than=2022-05-19T13%3A28%3A18.895833554%2B0>
2022/05/19 17:53:42.916517 66027#118 [debug] QueryLog: prepared data (0/541) older than 2022-05-19 13:28:18.895833554 +0200 CEST in 0s
2022/05/19 17:53:42.916517 66027#118 [debug] GET /control/querylog?search=&response_status=blocked_parental&older_than=2022-05-19T13%3A28%3A18.895833554%2B0>
2022/05/19 17:53:42.916517 66027#118 [debug] QueryLog: prepared data (0/541) older than 2022-05-19 13:28:18.895833554 +0200 CEST in 0s
2022/05/19 17:53:42.916517 66027#118 [debug] GET /control/querylog?search=&response_status=blocked_parental&older_than=2022-05-19T13%3A28%3A18.895833554%2B0>
2022/05/19 17:53:42.916517 66027#118 [debug] QueryLog: prepared data (0/541) older than 2022-05-19 13:28:18.895833554 +0200 CEST in 0s
2022/05/19 17:53:43.916517 66027#118 [debug] GET /control/querylog?search=&response_status=blocked_parental&older_than=2022-05-19T13%3A28%3A18.895833554%2B0>
2022/05/19 17:53:43.916517 66027#118 [debug] QueryLog: prepared data (0/541) older than 2022-05-19 13:28:18.895833554 +0200 CEST in 0s
2022/05/19 17:53:43.916517 66027#118 [debug] GET /control/querylog?search=&response_status=blocked_parental&older_than=2022-05-19T13%3A28%3A18.895833554%2B0>
2022/05/19 17:53:43.916517 66027#118 [debug] QueryLog: prepared data (0/541) older than 2022-05-19 13:28:18.895833554 +0200 CEST in 0s
2022/05/19 17:53:43.916517 66027#118 [debug] GET /control/querylog?search=&response_status=blocked_parental&older_than=2022-05-19T13%3A28%3A18.895833554%2B0>
2022/05/19 17:53:43.916517 66027#118 [debug] QueryLog: prepared data (0/541) older than 2022-05-19 13:28:18.895833554 +0200 CEST in 0s
2022/05/19 17:53:43.916517 66027#118 [debug] GET /control/querylog?search=&response_status=blocked_parental&older_than=2022-05-19T13%3A28%3A18.895833554%2B0>
2022/05/19 17:53:43.916517 66027#118 [debug] QueryLog: prepared data (0/541) older than 2022-05-19 13:28:18.895833554 +0200 CEST in 0s
2022/05/19 17:53:43.916517 66027#974 [debug] github.com/AdguardTeam/dnsproxy/proxy.(*Proxy).udpHandlePacket(): Start handling new UDP packet from 172.16.1.1>
2022/05/19 17:53:43.916517 66027#974 [debug] github.com/AdguardTeam/dnsproxy/proxy.(*Proxy).logDNSMessage(): IN: ;; opcode: QUERY, status: NOERROR, id: 60197
;; flags: rd; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 0

Maybe the problem is that it is searching for data older than 2022-05-19 13:28:18.895833554 +0200 CEST ...at this time there were no entries. I try to look tomorrow again.

**Reply by ainar-g** [reply]
It looks like the frontend keeps requesting the same thing over and over.  I think we've fixed a similar issue in the edge version.  Could you please check if the edge version has that problem?  You can download it [here][1].  Don't forget to backup AGH data.

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/Platforms#releases

**Reply by ElBarto2022** [reply]
Tried the edge Version (v0.108.0-a.128+79d85a24) and still having the problem. :( 
I think the querylog.json is the problem. After it is generated the view of different protocols starts to hang.

**Reply by ainar-g** [reply]
@ElBarto2022, what OS are you running?

Also, if the issue is still there, are you comfortable with sending us your `querylog.json` (or any querylog.json` with which the issue is reproduced at devteam@adguard.com?  You could replace yours with a new one, try reproducing the issue, and send the new one to us.  Thanks.

**Reply by ElBarto2022** [reply]
It is a pfsense 2..6.0 enviroment with FreeBSD 12.3-STABLE. The json file is growing over days to hunderts of MB. I send you the file which was new created after i recognized that it is hanging again. Using Edge v0.108.0-a.130+c0ac82be
Maybe it is the combination of the Filterlist  https://raw.githubusercontent.com/StevenBlack/hosts/master/alternates/gambling-porn-social/hosts and the adguard parental control web service option

**Reply by ainar-g** [reply]
@ElBarto2022, thank you very much, we're able to reproduce that on a Linux system as well.  We'll try to fix it in #4591.
